src/js/components/Select/stories/OptionsFromObject.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Box, Select } from 'grommet';
+
+const objectOptions = [
+  {
+    lab: 'Male',
+    val: 1,
+  },
+  {
+    lab: 'Female',
+    value: 2,
+  },
+  {
+    lab: 'Non Binary',
+    value: 3,
+  },
+  {
+    lab: 'Other',
+    value: 4,
+  },
+];
+
+const OptionsFromObject = () => {
+  const [value, setValue] = useState('');
+
+  return (
+    <Box fill align="center" justify="start" pad="large">
+      <Select
+        id="select"
+        name="select"
+        placeholder="Select"
+        labelKey="lab"
+        value={value}
+        options={objectOptions}
+        onChange={({ option }) => setValue(option)}
+      />
+    </Box>
+  );
+};
+
+storiesOf('Select', module).add('OptionsFromObject', () => (
```

Comment:
Hey there,

thanks for this story. can you change the label here with spaces? this is publicly available on storybook so it should be easily readable. Maybe Object options would also fit better. other wise good to go

Reply:
```suggestion
storiesOf('Select', module).add('Options from object', () => (
```